app/models/nearby_facility.rb

```diff
@@ -36,13 +42,25 @@ def request_isochrone(waypoint, params)
         key: Settings.bing.key
       }
 
-      response = Faraday.get "#{Settings.bing.base_api_url}/Isochrones", query
+      response = Faraday.get "#{Settings.bing.base_api_url}/Routes/Isochrones", query
```

Comment:
You moved `/Routes` here from the settings.yml file (which makes sense to me) -- but if production/staging doesn't have a corresponding URL then this will turn into `/Routes/Routes`

Reply:
thanks, will check that